### DISCUSSION TITLE
Failed Config for Emporia_Vue - Fixed

### DISCUSSION THREAD

**Discussion by loca5790** [emporia-vue-local/esphome#61] [General]
Hey All,

Just updated my windows computer to windows 11 due to circumstances outside of my control.

I had to redo my entire computer including python and esphome.

I am trying to update my emporia vue wirelessly like I have done plenty of times before but now am getting this error referenced...

Failed config

external_components:

"esphome:
  name: emporiavue2-main

external_components:
  - source: github://flaviut/esphome@emporia-vue
    components: [ emporia_vue ]"

it's saying git is not installed but required for external_components.

I'm sure it's due to an esphome update and something simple.

should have spent a little more time..... got it fixed apparently I didn't restart.

**Comment by flaviut** [comment]
[Install git](https://git-scm.com/download/win) and make sure it's in your
PATH.

**Reply by loca5790** [reply]
appreciate you, sorry for wasting your time but it was 100% a git issue.

**Reply by flaviut** [reply]
no problem at all! there's probably a dozen other people who will run into this issue in the future too & find this solution.
